Wireframe/index.html

```diff
@@ -1,13 +1,75 @@
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Wireframe</title>
+  <link rel="stylesheet" href="./css/style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Onest:wght@100;300;500;700;900&family=Outfit:wght@200;400;600;800&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <!-- Add your HTML markup here -->
+  <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+
+  <header>
+    <h1>GitPlatform</h1>
+    <h4>These website is for new developers. Here you can learn more about Git</h4>
+  </header>
+
+  <main>
+    <section class="fC" style="background-image: url(./images/office.jpg);">
+      <article>
+        <h3 class="firstArticle">What is Git?</h3>
+        <div>
+          <p>Git is a version control system for software development. It tracks code changes, manages
+            repositories, and enables effective collaboration. Developers use commands like commit, branch, and merge to
+            control code versions and histories. Git's decentralization allows offline work and enhances data security.
+            Hosting platforms like GitHub facilitate remote repositories and teamwork. It's an essential tool, ensuring
+            organized, transparent, and cooperative software development.</p>
+          <button>READ MORE</button>
+        </div>
+      </article>
+    </section>
+    <section class="fS" style="background-image: url(./images/developer.jpg);">
+      <article>
+        <h3>Why do developers need Git?</h3>
+        <div>
+          <p>Developers need Git to track and manage code changes, collaborate with others, maintain version history,
+            and
+            ensure code stability, making software development more efficient and organized.</p>
+          <button>READ MORE</button>
+        </div>
+      </article>
+    </section>
+    <section class="fT" style="background-image: url(./images/branch.jpg);">
+      <article>
+        <h3>What is a branch in Git?</h3>
+        <div>
+          <p>A branch in Git is a separate line of code development that diverges from the main codebase, enabling
+            developers to work on features or fixes independently before merging changes.</p>
+          <button>READ MORE</button>
+        </div>
+      </article>
+    </section>
+  </main>
+<hr>
+  <footer>
+    <h6>FOOTERS USUALLY CONTAIN CONTACT LINKS AND OTHER INFORMATION ABOUT THE SITE OVERAL</h6>
```

Comment:
Semantically, using a heading element `h6` for a footer is not recommended. Instead, styling can be achieved with CSS.